packages/ignite/src/plugins/default/commands/add.js

```diff
@@ -42,20 +42,23 @@ module.exports = async function (context) {
   // take the last parameter (because of https://github.com/infinitered/gluegun/issues/123)
   // prepend `ignite` as convention
   const moduleName = `ignite-${parameters.array.pop()}`
-  info(`🔎    Finding Plugin`)
+  info(`🔎    Finding ${moduleName} on npmjs.com`)
   const moduleExists = await Exists(moduleName)
   // it exists?  Let's install it else warn
   if (moduleExists) {
-    success(`${checkmark}    Installing`)
+    info(`${checkmark}    Installing npm module`)
 
     if (ignite.useYarn) {
-      Shell.exec(`yarn add ${moduleName} --dev`, {silent: true})
+      Shell.exec(`yarn add ${moduleName}`, {silent: true})
```

Comment:
We should change this to also support the config flags.  Should be dev dep by default.